src/Outline.tsx

```diff
@@ -32,9 +32,7 @@ export type OutlineProps = {
 } & EventProps<PDFOutline | null | false | undefined>;
 
 export default function Outline(props: OutlineProps) {
-  const context = useContext(DocumentContext);
-
-  invariant(context, 'Unable to find Document context. Did you wrap <Outline /> in <Document />?');
```

Comment:
Do we want to add this part of the message back?

```
Did you wrap <Outline /> in <Document />?
```

Reply:
Yes, very much :D